src/services/localizations/assertValidFileContent.ts

```diff
@@ -0,0 +1,17 @@
+import * as Joi from 'joi';
+
+// The service actually accepts almost any key, but we want to enforce a pattern
+const keyPattern = /^[a-z0-9_.-]{1,200}$/i;
+
+const localizationFileSchema = Joi.object()
+  .pattern(keyPattern, Joi.string());
+
+export function assertValidFileContent(fileName: string, content: unknown): content is Record<string, string> {
```

Comment:
I respect the grind with the predicate  `content is Record<string, string>` but it doesn't actually do anything in its use right? 😓 

I've added an incredibly quirky comment where it's used (Don't hit me....)

Reply:
Yikes, I've made this mistake many times. I belief at some point is was not possible but there is a `asserts content is Record<string, string>` for some time now. Which I meant to use here. Fixed!